build-system/pr-check/experiment-build.js

```diff
@@ -32,7 +32,6 @@ const {runCiJob} = require('./ci-job');
 const jobName = `${experiment}-build.js`;
 
 /**
- * @return {void}
  */
```

Comment:
nit: remove `/** */` since there's no more comment

Reply:
I plan to enable JSDoc enforcement soon to stem the amount of undocumented functions being added and removing these would slow that down. I could try to write an actual description of each of these functions or else I could add TODOs in each of these places, but both of those seem like pretty significant changes.

Reply:
I think the main objection is in having a blank JSDoc. When you add the check, what will this end up being?

Also, is enforcement applicable only to those functions where there ought to be a non-empty JSDoc? Or will it insist on a blank `/** */`? (We should try and avoid this.)

Reply:
I definitely understand the objection but yes, it will insist on at least a blank comment. Given that there are 84 occurrences of this at them moment I would like to remove all these for now and try to add them all back in a better way in another dedicated PR.